stable/taxi-demo/templates/demo-function.yaml

```diff
@@ -7,12 +7,16 @@ apiVersion: nuclio.io/v1beta1
 kind: Function
 metadata:
   name: {{ $val.name }}
+  labels:
+    nuclio.io/project-name: taxi-demo
 spec:
   handler: {{ $val.handler }}
   image: {{ $val.image.repository }}:{{ $val.image.tag }}
   maxReplicas: 1
   minReplicas: 1
   runtime: {{ $val.runtime }}
+  build:
+    functionSourceCode: {{ $val.functionSourceCode }}
```

Comment:
Use something like:
```
functionSourceCode: {{ $val.functionSourceCode | b64enc | quote }}
```

This will require `values.yaml` file to have the actual source code